sample_sensor_kit_launch/launch/pointcloud_preprocessor.launch.py

```diff
@@ -26,28 +26,82 @@
     )
 
+    # switch between sync_and_concatenate_filter and synchronizer_filter
+    if not separate_concatenate_node_and_timesync_node:
+        sync_and_concat_component = ComposableNode(
+            package="pointcloud_preprocessor",
+            plugin="pointcloud_preprocessor::PointCloudConcatenateDataSynchronizerComponent",
+            name="sync_and_concatenate_filter",
+            remappings=[
+                ("~/input/twist", "/sensing/vehicle_velocity_converter/twist_with_covariance"),
+                ("output", "concatenated/pointcloud"),
+            ],
+            parameters=[
+                {
+                    "input_topics": [
+                        "/sensing/lidar/top/outlier_filtered/pointcloud",
+                        "/sensing/lidar/left/outlier_filtered/pointcloud",
+                        "/sensing/lidar/right/outlier_filtered/pointcloud",
+                    ],
+                    "output_frame": LaunchConfiguration("base_frame"),
+                    "approximate_sync": True,
+                    "publish_synchronized_pointcloud": True,  # set true if you want to publish synchronized pointcloud
+                }
+            ],
+            extra_arguments=[{"use_intra_process_comms": LaunchConfiguration("use_intra_process")}],
+        )
+        concat_components = [sync_and_concat_component]
+    else:
+        time_sync_component = ComposableNode(
+            package="pointcloud_preprocessor",
+            plugin="pointcloud_preprocessor::PointCloudDataSynchronizerComponent",
+            name="synchronizer_filter",
+            remappings=[
+                ("~/input/twist", "/sensing/vehicle_velocity_converter/twist_with_covariance"),
+            ],
+            parameters=[
+                {
+                    "input_topics": [
+                        "/sensing/lidar/top/outlier_filtered/pointcloud",
+                        "/sensing/lidar/left/outlier_filtered/pointcloud",
+                        "/sensing/lidar/right/outlier_filtered/pointcloud",
+                    ],
+                    "output_frame": LaunchConfiguration("base_frame"),
+                    "approximate_sync": True,
+                }
+            ],
+            extra_arguments=[{"use_intra_process_comms": LaunchConfiguration("use_intra_process")}],
+        )
+
+        concat_component = ComposableNode(
+            package="pointcloud_preprocessor",
+            plugin="pointcloud_preprocessor::PointCloudConcatenationComponent",
+            name="concatenate_filter",
+            remappings=[("output", "points_raw/concatenated")],
+            parameters=[
+                {
+                    "input_topics": [
+                        "/sensing/lidar/top/outlier_filtered/pointcloud_synchronized",
+                        "/sensing/lidar/left/outlier_filtered/pointcloud_synchronized",
+                        "/sensing/lidar/right/outlier_filtered/pointcloud_synchronized",
```

Comment:
`/sensing/lidar/top/synchronized/pointcloud` sounds better to me, like other nodes output

Reply:
Ahh...  I do want, but not urgent.
Defining like ~/synchronized/pointcloud in codes, and passing (or remapping) actual topic name from launcher is almost autoware coding manner.